x-pack/plugins/licensing/public/plugin.ts

```diff
@@ -134,6 +134,7 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup, LicensingPl
     }
     return {
       refresh: this.refresh,
+      getLicense: async () => await firstValueFrom(this.license$!.pipe(take(1))),
```

Comment:
NIT the `take(1)` is unnecessary by [firstValueFrom definition](https://rxjs.dev/api/index/function/firstValueFrom)

Reply:
good call! I'll remove the pipe

Reply:
Addressed in [`1f71fe3` (#194499)](https://github.com/elastic/kibana/pull/194499/commits/1f71fe3cd7698e213e463bc2a151647bc2f438a2)